recipes/clinker/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "clinker" %}
+{% set version = "1.31" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: "v{{ version }}.tar.gz"
+  url: https://github.com/Oshlack/Clinker/archive/v{{ version }}.tar.gz
+  md5: "5487f99af6b9499c87d4d945e9cd9790"
+
+build:
+  number: 0
+
+requirements:
+  run:
+    - samtools
+    - bpipe
+    - star 2.5.3a.*
+    - python 2.7.*
+
+
+
+
+test:
+  commands:
+    - bpipe --help
```

Comment:
This does not look like a clinker test :)

Reply:
Clinker is invoked via bpipe - but we may need to write a wrapper.